cheatsheets/Authentication_Cheat_Sheet.md

```diff
@@ -23,49 +23,24 @@ For information on validating email addresses, please visit the [input validatio
 
 A key concern when using passwords for authentication is password strength. A "strong" password policy makes it difficult or even improbable for one to guess the password through either manual or automated means. The following characteristics define a strong password:
 
-### Password Length
+- Password Length
+    - **Minimum** length of the passwords should be **enforced** by the application. Passwords **shorter than 8 characters** are considered to be weak ([NIST SP800-63B](https://pages.nist.gov/800-63-3/sp800-63b.html)). 
+    - **Maximum** password length should not be set **too low**, as it will prevent users from creating passphrases. Typical maximum length is 128 characters.
 
-Longer passwords provide a greater combination of characters and consequently make it more difficult for an attacker to guess.
+- Do do not truncate passwords. Make sure that every character the user types in is actually included in the password. 
```

Comment:
In order to prevent [DOS using password length](https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Password_Storage_Cheat_Sheet.md#input-password-size) then the password can be hashed with SHA-512 before to be passed to the final Hashing algorithm (Argon2, Bcrypt...). I think that it can be interesting to add a remark on that point.

Reply:
No need. The 128 password length restriction should be enough. Doing hashing before the hash mechanisms doesn't sound that sane.

Reply:
OK thank for the info 👍